### DISCUSSION TITLE
Arduino_GFX有沒有從外部加載字庫的方法？

### DISCUSSION THREAD

**Discussion by ham0nd** [moononournation/Arduino_GFX#246] [Q&A]
如題，我嘗試編譯左一輪`U8g2FontUTF8Chinese.ino`這個例子，但是發現默認esp32芯片分區配置的情況下幾乎占滿（如圖），這導致我無法有足夠的flash空間黎添加其他功能。我發現`TFT_eSPI`庫中有從sd卡中`load_fonts`的功能，想問一下老師Arduino_GFX有沒有類似從外部加載字庫的方法？這樣就能省番滴珍貴的flash空間了，謝謝
![image](https://user-images.githubusercontent.com/43801159/209501343-2ad74f34-1029-44ab-b8ca-e840cacb3937.png)

**Comment by moononournation** [comment]
you can use reduced set
